layouts/error.vue

```diff
@@ -1,22 +1,25 @@
 <template>
   <article v-if="error.statusCode === 404" class="detail-layout">
     <h1>We kunnen de pagina die je zoekt niet vinden</h1>
-    <div class="help-block">
-      <h2>Hoe kunnen we je verder helpen?</h2>
-      <div class="help-block-section">
-        <h3>1. Verder navigeren vanaf hier?</h3>
-        <p>
-          Dat kan op verschillende manieren:
-        </p>
-        <ul class="icon-list">
-          <li>
-            <i class="icon-home" aria-hidden="true" /><span>Terugkeren naar de
-              <nuxt-link :to="'/'">homepage</nuxt-link></span>
-          </li>
-          <li>
-            <i class="icon-fancyback" aria-hidden="true" /><span>De terugknop van je browser gebruiken</span>
-          </li>
-        </ul>
+    <div class="help-block highlight">
+      <div class="highlight__inner">
+        <div>
+          <h2>Hoe kunnen we je verder helpen?</h2>
```

Comment:
How will we translate these texts?

Reply:
We won't. It's not a multi lingual site.